apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text, Separator } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { type BagTitleType } from './BagTypes';
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Bag = {|
+  +type: BagTitleType,
+  +quantity: number,
+  +dimensions: string,
+  +weight: string,
+|};
+
+type Props = {|
+  +price: ?string,
+  +bags: Array<Bag>,
+  +bagIndex: number,
+  +onToggleCheck: (BagTitleType, number) => void,
+  +type: string,
+  +currentlyChosen: number,
+|};
+
+export default function BaggageBundle({
+  currentlyChosen,
+  type,
+  bagIndex,
+  bags,
+  price,
+  onToggleCheck,
+}: Props) {
+  const shouldBeChecked = currentlyChosen === bagIndex;
+  function handleToggleCheck() {
+    onToggleCheck(type, bagIndex);
+  }
+
+  return (
+    <View
+      style={[styles.container, shouldBeChecked && styles.containerChecked]}
+    >
+      <View style={styles.row}>
+        <RadioButton
+          type="check"
+          checked={shouldBeChecked}
+          onPress={handleToggleCheck}
+        />
+        <View style={styles.baggageDetails}>
+          {bags.map((bag, index) => {
+            return (
+              <BaggageTypeDetail
+                key={index}
+                type={bag.type}
+                quantity={bag.quantity}
+                dimensions={bag.dimensions}
+                weight={bag.weight}
+                chosen={shouldBeChecked}
+              />
+            );
+          })}
+          <Separator />
+        </View>
+      </View>
+      <Text>{price}</Text>
+    </View>
+  );
+}
+
+const borderWidth = 0.5;
+const checkedBorderWidth = 2;
+const borderWidthDiff = checkedBorderWidth - borderWidth;
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: parseFloat(defaultTokens.spaceXSmall),
+    marginTop: parseFloat(defaultTokens.spaceSmall),
+    marginHorizontal: parseFloat(defaultTokens.spaceSmall),
+    marginBottom: parseFloat(defaultTokens.spaceXXXSmall),
+    borderWidth,
+    borderStyle: 'solid',
+    borderColor: defaultTokens.borderColorTableCell,
+    borderRadius: 4,
+    shadowColor: defaultTokens.paletteInkDark,
```

Comment:
I think that whole element should have white background to have shadow visible only around edges.

Left: design | Right: iOS build
<img width="517" alt="Screenshot 2019-03-18 at 12 05 36" src="https://user-images.githubusercontent.com/2660330/54526393-ef397700-4976-11e9-8060-96ca3b8ba7b5.png">

Reply:
Thanks! Looks much better now